specs/olm.spec

```diff
@@ -48,4 +48,9 @@ Importance: Critical
 Uninstalls `openshift-pipelines` operator using olm
 Steps:
   * Validate Operator should be installed
-  * Uninstall Operator
\ No newline at end of file
+  * Uninstall Operator
+
+## Verify that Pipelines operator installs quickstarts: ID PIPELINES-09-TC06
+Tags: install, upgrade
```

Comment:
I'm sorry...I forgot about other metadata, please add
```
Component: Operator
Level: Integration
Type: Functional
Importance: Critical
```
similarly to lines 43-46